workshops/models.py

```diff
@@ -569,7 +569,8 @@ def get_invoice_form_url(self):
             'entry.351294200': 'Workshop administrative fee',  # Reason
 
             # Date of event
-            'entry.1749215879': '{:%Y-%m-%d}'.format(self.start),
+            'entry.1749215879': ('{:%Y-%m-%d}'.format(self.start)
+                                 if self.start else ''),
```

Comment:
I'd create a helper function format_date in util.py and call it with `format_date(self.start)`, then look for other places where dates are formatted.